src/test/java/org/openrewrite/java/logging/jboss/JBossLoggingBestPracticesTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Moderne Source Available License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://docs.moderne.io/licensing/moderne-source-available-license
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.logging.jboss;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class JBossLoggingBestPracticesTest implements RewriteTest {
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipeFromResource(
+          "/META-INF/rewrite/jboss.yml",
+          "org.openrewrite.java.logging.jboss.JBossLoggingBestPractices"
+        );
+    }
+
+    @DocumentExample
+    @Test
+    void convertInfo() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              import org.jboss.logging.Logger;
+
+              class Test {
+                  void test(Logger logger, String msg, Object o) {
+                      logger.info(msg, new Object[]{o});
+                  }
+              }
+              """,
+            """
+              import org.jboss.logging.Logger;
+
+              class Test {
+                  void test(Logger logger, String msg, Object o) {
+                      logger.infov(msg, o);
```

Comment:
It's subtle, but this chains back to back two recipes to go from `info` to `infov` and from that object array to just the arguments taken from it.